src/window.rs

```diff
@@ -202,6 +202,36 @@ impl Window {
         self.window.get_inner_size()
     }
 
+    /// Returns the size in points of the client area of the window.
+    ///
+    /// The client area is the content of the window, excluding the title bar and borders.
+    /// To get the dimensions of the frame buffer when calling `glViewport`, multiply with hidpi factor.
+    ///
+    /// Returns `None` if the window no longer exists.
+    ///
+    /// DEPRECATED
+    #[inline]
```

Comment:
I didn't mention it, but it should have a `#[deprecated]` attribute so that a compiler warning is shown when you use them.

Reply:
added